### PR TITLE
Wiki Article: How to Improve Someone's else pull request

### PR DESCRIPTION
**Pull request by Rafase282**
Creates article on how mods can update or improve someone's PR

### CODE REVIEW

File: Wiki-Improve-PR.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# How to take someone's PR and make your own base dof it whilie keeping his commit
```
Comment (theflametrooper) [inline Wiki-Improve-PR.md]: ~~whilie~~ while

~~his~~ their

**Comment by alayek** [comment]
# LGTM :+1:

File: Wiki-Improve-PR.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
+# How to take someone's PR and make your own base dof it while keeping thier commit
```
Comment (theflametrooper) [inline Wiki-Improve-PR.md]: ~~thier~~ their

**Comment by Rafase282** [comment]
Made all improvements

**Comment by theflametrooper** [comment]
## LGTM :+1:

Thanks @Rafase282 !!

**Pull request merged**